packages/@aws-cdk/aws-ecs-patterns/test/ec2/integ.multiple-application-load-balanced-ecs-service.expected.json

```diff
@@ -943,9 +943,11 @@
         "Conditions": [
           {
             "Field": "path-pattern",
-            "Values": [
-              "a/b/c"
-            ]
+            "PathPatternConfig": {
```

Comment:
In order to not cause unnecessary changes in all our user's templates, can we make sure that if the configuration can be expressed in the "legacy" format, that we do so?

Otherwise everyone around the world will freak out if all their templates change after a CDK update.

Reply:
I overlooked that problem. I'm agreed too.